layouts/shortcodes/cve-feed.html

```diff
@@ -1,5 +1,6 @@
 <table class="security-cves">
-  <caption>{{ T "cve_table" }}</caption>
+  {{ $feed := getJSON .Site.Params.cveFeedBucket }}
+  <caption>{{ T "cve_table" }} (updated on {{ $feed._kubernetes_io.updated_at | time.Format "02 Jan 2006 at 15:04:05 MST" }})</caption>
```

Comment:
Please make string “updated on” localizable.

Reply:
Yes! And I should clean up the no longer used localization strings.

Reply:
Can we replace "updated on" with "last updated: " ?

Reply:
Done here https://github.com/kubernetes/website/pull/38579/commits/7f13b94edfc3302f351df03967bb2ab4edf3cfbc#diff-2d1b915ccd9669900a511913db7cc235638bbd7916ba066742477288fd0c503cR3.